.github/workflows/build-release.yml

```diff
@@ -0,0 +1,75 @@
+name: Release
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+
+  release_amd64:
+    name: Release (AMD64 - CentOS 7 Target)
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Build code-server Binary
+        run: yarn && bash ./scripts/ci.bash    
+        env:
+           MAJOR_VERSION: '2'
+           TARGET: 'linux'
+           VERSION: '${MAJOR_VERSION}.${GITHUB_SHA}'
+           VSCODE_VERSION: '1.38.1'
+           MINIFY: 'true' 
+           PACKAGE: 'true'
+           
+      - uses: ncipollo/release-action@v1
```

Comment:
Would it be a good idea to add `draft: true` to these? I'd like to review the release before it actually gets published since a bunch of people will get emailed about it as soon as it gets made.